lib/core/collections.toit

```diff
@@ -2456,6 +2456,13 @@ class Set extends HashedInsertionOrderedCollection_ implements Collection:
     do: if not predicate.call it: return false
     return true
 
+  /**
+  Copies the set.
+  The set is shallow.
```

Comment:
This isn't an awesome comment.

Maybe a slightly better formatted comment that states something about the returned set would be nicer?

Reply:
done.